config/routes.rb

```diff
@@ -1,10 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Reveal health status on /up that returns 200 if the app boots with no exceptions, otherwise 500.
-  # Can be used by load balancers and uptime monitors to verify that the app is live.
-  get "up" => "rails/health#show", as: :rails_health_check
-
-  # Defines the root path route ("/")
-  # root "posts#index"
-end
+  root "users#index"
+  resources :users, only: [:index, show] do
+    resources :posts, only: [:index, show]
+  end
```

Comment:
- [ ] You missed adding the colon `:` to show property. Please make sure to fix this so the routes work as expected.
  - You can consider using the below code.
  
```ruby
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end